bpf/lib/fib.h

```diff
@@ -14,7 +14,8 @@
 static __always_inline int
 redirect_direct_v6(struct __ctx_buff *ctx __maybe_unused,
 		   int l3_off __maybe_unused,
-		   struct ipv6hdr *ip6 __maybe_unused, int *oif)
+		   struct ipv6hdr *ip6 __maybe_unused,
+		   int iif __maybe_unused, int *oif)
```

Comment:
the changes in this file is not part of original PR, do we need to backport the dependent PR ?

Reply:
ah got it, thanks :100: